sample/shared/src/messageTypes.ts

```diff
@@ -0,0 +1,8 @@
+import { useEventBusDispatcher, useEventBusListener } from "@squide/react-router";
+
+export type MessageTypes =
+    "write-to-host"
+;
+
+export const useApplicationEventBusDispatcher = useEventBusDispatcher<MessageTypes>;
```

Comment:
This is a pattern stolen from redux
 https://redux-toolkit.js.org/tutorials/typescript#define-typed-hooks